Konard/Integrations/Children of the Dragon/race-dragonborn-revised.xml

```diff
@@ -0,0 +1,428 @@
+		</rules>
+	</element>
+	<element name="Draconic Fear" type="Racial Trait" source="Children of the Dragon" id="ID_DMSG_COTD_RACIAL_TRAIT_DRACONIC_FEAR">
+		<description>
+			<p>You can roar as an action, forcing each creature of your choice within 30 feet of you to make a Wisdom saving throw (DC 8 + your proficiency bonus + your Charisma modifier) if it can see and hear you. On a failed save, the target is frightened of you for 1 minute. The frightened target can repeat the saving throw at the end of each of its turns, ending the effect on itself on a success. You regain the use of this trait after finishing a long rest.</p>
+		</description>
+		<sheet>
+			<description>You roar forcing each creature of your choice within 30 feet of you to be frightened for 1 minute. (wis. save) DC {{draconic-fear:dc}}</description>
+		</sheet>
+		<rules>
+			<stat name="draconic-fear:dc" value="8" />
+			<stat name="draconic-fear:dc" value="charisma:modifier" />
+			<stat name="draconic-fear:dc" value="proficiency" />
+		</rules>
+	</element>
+	
+	<element name="Murkdweller Dragonborn" type="Sub Race" source="Children of the Dragon" id="ID_DMSG_COTD_SUB_RACE_MURKDWELLER">
+		<supports>Dragonborn</supports>
+		<description>
+			<p>Murkdwellers are the only ones with a tail, and are often shunned because of it. Even while half dragons, known for their long tails, are generally accepted within dragonborn society.</p>
+			<p class="indent"><strong><em>Darkvision.</em></strong> You can see in dim light within 60 feet of you as if it were bright light, and in darkness as if it were dim light. You can’t discern color in darkness, only shades of gray.</p>
+			<p class="indent"><strong><em>Tail Lash.</em></strong> Your tail can be used to push or trip up opponents. You can use your bonus action to shove a creature within 5 feet of you. You can use this trait a number of times equal to your Strength modifier (minimum of 1). You regain all expended uses when you finish a long rest.</p>
+		</description>
+		<sheet display="false"/>
+		<rules>
+			<grant type="Vision" id="ID_VISION_DARKVISION" />
+			<grant type="Racial Trait" id="ID_DMSG_COTD_RACIAL_TRAIT_TAIL_LASH" />
+		</rules>
+	</element>
+	<element name="Tail Lash" type="Racial Trait" source="Children of the Dragon" id="ID_DMSG_COTD_RACIAL_TRAIT_TAIL_LASH">
+		<description>
+			<p>Your tail can be used to push or trip up opponents. You can use your bonus action to shove a creature within 5 feet of you. You can use this trait a number of times equal to your Strength modifier (minimum of 1). You regain all expended uses when you finish a long rest.</p>
+		</description>
+		<sheet action="Bonus Action" usage="{{strength:modifier}}/Long Rest">
+			<description>Your tail can be used to push or trip up opponents. You can use your bonus action to shove a creature within 5 feet of you. You can use this trait {{tail-lash:uses}} times. You regain all expended uses when you finish a long rest.</description>
```

Comment:
```suggestion
			<description>Your tail can be used to push or trip up opponents. You can use your bonus action to shove a creature within 5 feet of you. </description>
```

Info redundant since info is above